app/models/migration/ecf/participant_outcome/npq.rb

```diff
@@ -0,0 +1,7 @@
+module Migration::Ecf
+  class ParticipantOutcome::Npq < BaseRecord
```

Comment:
we should probably bring across the participant outcome api requests as well, will add a ticket